### PR TITLE
Add new rule package_systemd-journal-remote_installed

### PR DESCRIPTION
**Pull request by dodys**
#### Description:

- Rule to install package `systemd-journal-remote`

#### Rationale:

- This rule is needed on CIS for both Ubuntu and RHEL

### CODE REVIEW

File: controls/cis_rhel8.yml
Comment (marcusburghardt) [inline controls/cis_rhel8.yml:1750]:
```suggestion
    status: manual
```
We should keep the status aligned to the requirement, in case of CIS, even if we have a rule for this. In any case, I believe it is safe to keep this rule listed there since it will just install the package. The `4.2.2.1.2` is more complicated to have an automated rule.

**Review by marcusburghardt — CHANGES_REQUESTED** [review]

File: controls/cis_rhel8.yml
Comment (dodys) [reply controls/cis_rhel8.yml:1750]: @marcusburghardt  I think you will want to check 4.2.1.6 or other rules that have Manual in the title but status is automated, there are quite a few

Comment (marcusburghardt) [reply controls/cis_rhel8.yml:1750]: Correct. It should also be updated to `status: manual` in `4.2.1.6`. Thanks for noticing this. I didn't find other cases like this. Would you mind to update in this PR? Otherwise I can create a separate one.

File: linux_os/guide/system/logging/journald/package_systemd-journal-remote_installed/rule.yml
Comment (marcusburghardt) [inline linux_os/guide/system/logging/journald/package_systemd-journal-remote_installed/rule.yml:17]: The CI tests are also complaining about missing `cce` for `rhel8`.

File: controls/cis_rhel8.yml
Comment (dodys) [reply controls/cis_rhel8.yml]: If that's okay, I will leave that to you in another PR with any CCE changes you need.

Comment (dodys) [reply controls/cis_rhel8.yml]: just dropped the rhel bits altogether from this PR.

**Review by marcusburghardt — APPROVED** [review]
Thanks

**Comment by marcusburghardt** [comment]
Overriding the CODEOWNERS requirement as @dodys can't approve this PR.

**Pull request merged**
